### ISSUE TITLE
mate-power-manager doesn't turn off the display despite elapsed timeout

### ISSUE THREAD

**Issue by raveit65** [mate-desktop/mate-power-manager#71]
Orginal report at fedora bugzilla
https://bugzilla.redhat.com/show_bug.cgi?id=994232

Description of problem:
MATE Power Manager doesn't turn off the display despite elapsed timeout. The screensaver kicks in, but the display is never turned off.

Version-Release number of selected component (if applicable):
mate-power-manager-1.6.1-3.fc19.x86_64

How reproducible:
Always

Steps to Reproduce:
1. Set "Put display to sleep when inactive for" to 1 minute in Power Management Preferences, "On AC Power" tab.
2. Wait 1 minute without touching the keyboard or mouse.

Actual results:
Nothing happens.

Expected results:
Display should be turned off.

Additional info:
Similar symptom as in bug 949070, but I get this on both a desktop PC and a laptop.

$ grep -i dpms /var/log/Xorg.0.log
[     3.607] Initializing built-in extension DPMS
[     3.754](II) RADEON(0): DPMS capabilities: StandBy Suspend Off
[     3.831](==) RADEON(0): DPMS enabled

xset dpms force off does turn the display off.

$ rpm -qa *mate* |sort
imsettings-mate-1.6.3-1.fc19.x86_64
libmatekbd-1.6.1-1.fc19.x86_64
libmatekeyring-1.6.0-1.fc19.x86_64
libmateweather-1.6.1-1.fc19.x86_64
libmatewnck-1.6.1-1.fc19.x86_64
mate-applets-1.6.1-4.fc19.x86_64
mate-backgrounds-1.6.0-1.fc19.noarch
mate-calc-1.6.0-1.fc19.x86_64
mate-character-map-1.6.0-4.fc19.x86_64
mate-control-center-1.6.0-2.fc19.x86_64
mate-desktop-1.6.1-8.fc19.x86_64
mate-desktop-libs-1.6.1-8.fc19.x86_64
mate-dialogs-1.6.0-1.fc19.x86_64
mate-document-viewer-1.6.0-6.fc19.x86_64
mate-document-viewer-data-1.6.0-6.fc19.noarch
mate-document-viewer-libs-1.6.0-6.fc19.x86_64
mate-file-archiver-1.6.0-2.fc19.x86_64
mate-file-manager-1.6.1-9.fc19.x86_64
mate-file-manager-extensions-1.6.1-9.fc19.x86_64
mate-file-manager-open-terminal-1.6.0-1.fc19.x86_64
mate-icon-theme-1.6.1-1.fc19.noarch
mate-image-viewer-1.6.0-1.fc19.x86_64
mate-keyring-1.6.0-2.fc19.x86_64
mate-keyring-pam-1.6.0-2.fc19.x86_64
mate-media-1.6.0-2.fc19.x86_64
mate-menus-1.6.0-4.fc19.x86_64
mate-menus-libs-1.6.0-4.fc19.x86_64
mate-netspeed-1.6.0-1.fc19.x86_64
mate-notification-daemon-1.6.0-2.fc19.x86_64
mate-panel-1.6.0-3.fc19.x86_64
mate-panel-libs-1.6.0-3.fc19.x86_64
mate-polkit-1.6.0-1.fc19.x86_64
mate-power-manager-1.6.1-3.fc19.x86_64
mate-screensaver-1.6.0-1.fc19.x86_64
mate-sensors-applet-1.6.0-3.fc19.x86_64
mate-session-manager-1.6.0-3.fc19.x86_64
mate-settings-daemon-1.6.1-2.fc19.x86_64
mate-system-log-1.6.0-5.fc19.x86_64
mate-system-monitor-1.6.0-1.fc19.x86_64
mate-terminal-1.6.1-9.fc19.x86_64
mate-text-editor-1.6.0-2.fc19.x86_64
mate-themes-1.6.1-2.fc19.noarch
mate-utils-1.6.0-5.fc19.x86_64
mate-window-manager-1.6.2-3.fc19.x86_64

$ cat .xsession-errors
*\* (mate-power-manager:1062): WARNING **: levels is 0!

I can confirm this issue on a installation without consolekit (f19)

**Comment by Tho144** [comment]
I'm using Mate 1.26.0 , also experiencing unexpected behaviors with Power Manager (1.26.0) and XScreenSaver (6.06)
Power manager settings have NO AFFECT on putting display to sleep.
I also experienced: XScreenSaver "power management" being "randomly" turned off (checkbox "v" reverting to "empty")

I uninstalled Mate ScreenSaver long ago, exactly due to these behaviors, as per official XScreenSaver documentation recommendations.

So anyway, I switched XScreenSaver off and "killed" the daemon, yet power manager does not initiate display sleep.
I also checked dconf power manager settings, they seem to be consistent with required settings.

Also worth to mention that power manager "When laptop lid is closed" - does not blanking the screen - I can see the screen light still on, so - my solution until now was to lower screen brightness to 0% when leaving my laptop for expected long time like leaving home or going to bed.

For now, it seems like XScreenSaver settings are taking precedence, initiating the requested display sleep on set time, while power manager - "Put display to sleep" - is set to "never", but as this was feeling pretty "random", I have to keep an eye on this.
